src/components/ebay-toggle-button/index.marko

```diff
@@ -0,0 +1,52 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+
+static var ignoredAttributes = [
+    "class",
+    "title",
+    "subtitle",
+    "pressed"
+];
+
+<button
+    type="button"
+    class=[
+        "toggle-button",
+        input.gallery ? "toggle-button--gallery-layout" : "toggle-button--list-layout",
+        input.class
+    ]
+    aria-pressed=state.pressed && "true"
```

Comment:
Simplified the story to only show one controllable button instead of multiple inert ones. Should also be able to modify values in the `@img attributes` section for finer control.